servatrice/src/servatrice.h

```diff
@@ -143,6 +143,8 @@ public slots:
 	int getMaxMessageCountPerInterval() const { return maxMessageCountPerInterval; }
 	int getMaxMessageSizePerInterval() const { return maxMessageSizePerInterval; }
 	int getMaxGamesPerUser() const { return maxGamesPerUser; }
+    int getCommandCountingInterval() const { return commandCountingInterval; }
+    int getMaxCommandCountPerInterval() const { return maxCommandCountPerInterval; }
```

Comment:
The whole file is tab-indented, and i didn't want to pollute the commit. Other files in servatrice are tab-indented, i thought we'd prefer to address all these tab->spaces in an ad-hoc commit, if it's ok for you.